### PR TITLE
adds space and updates lifi link

### PR DESCRIPTION
**Pull request by goatsOnDrugs**
## Description
added space and updated link sent in discord

## Related Ticket

Resolves #<issue-number>

## Checklist

- [x] Building the site with `npm run build-site` works without errors
- [x] Building the app with `npm run build-app` works without errors
- [x] I formatted JS and TS files with running `npm run format-all`

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -13792,6 +13822,28 @@
...
+      "integrity": "sha512-kwVOfVoquufDQNmzzhGxji9A+/T+dhLKKOHEHldM1kMUoNzY6wwrt5H/Plnw4xbdE780kkwqJqHxjxoSYr706A==",
+      "requires": {
+        "@portabletext/types": "^1.0.3"
+      }
+    },
+    "@portabletext/types": {
+      "version": "1.0.3",
+      "resolved": "https://registry.npmjs.org/@portabletext/types/-/types-1.0.3.tgz",
+      "integrity": "sha512-SDDsdury2SaTI2D5Ea6o+Y39SSZMYHRMWJHxkxYl3yzFP0n/0EknOhoXcoaV+bxGr2dTTqZi2TOEj+uWYuavSw=="
+    },
```
Comment (Atmosfearful) [inline package-lock.json:13846]: these changes shouldnt be here plz remove

File: lib/constants/index.ts [hunk 2]
```diff
@@ -135,7 +135,7 @@ export const urls = {
   loveletter: "https://loveletter.klimadao.finance",
   mediaImage: "https://www.klimadao.finance/og-media.png",
   lifiStake:
-    "https://transferto.xyz/embed/stake-klima?fromChain=eth&fromToken=0x0000000000000000000000000000000000000000&toChain=pol&toToken=0x2791bca1f2de4661ed88a30c99a7a9449aa84174",
```
Comment (Atmosfearful) [inline lib/constants/index.ts:138]: nope there should be two,
`lifiStake` for /#/stake
and `lifiOffset` for /#/offset

**Review by Atmosfearful — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (goatsOnDrugs) [reply package-lock.json]: oops, sry removed

File: lib/constants/index.ts (hunk 2 above)
Comment (goatsOnDrugs) [reply lib/constants/index.ts:138]: added lifiOffset link (https://develop.transferto.xyz/embed/carbon-offset?fromChain=eth&fromToken=0x0000000000000000000000000000000000000000&toToken=0x2791bca1f2de4661ed88a30c99a7a9449aa84174) and added lifi stake back

**Review by 0xAeterno — CHANGES_REQUESTED** [review]
Leaving this related issue I created for consideration (https://github.com/KlimaDAO/klimadao/issues/630)

Part of the request was to mention Lifi and Etherspot on the /stake and /offset pages.
Though I'm not sure what's been agreed on as I haven't been part of further conversations after Trisync raised this.

FYI, LIFI and Etherspot have created a showcase page for both cross-chain staking and offsetting features.
The current links send users to a page to be embedded into an app.
As an outbound link from our app, I think it'll be a better experience to send users to the showcase page, where they have access to information on how the cross chain actions work, as well as a nicer UI to instil 'trust'.

See these pages here:
- [Staking](https://transferto.xyz/showcase/etherspot-klima?fromChain=eth&toChain=pol&toToken=0x2791bca1f2de4661ed88a30c99a7a9449aa84174)
- [Offsetting](https://transferto.xyz/showcase/carbon-offset?fromChain=eth&toChain=pol&toToken=0x2791bca1f2de4661ed88a30c99a7a9449aa84174)

What do you think?

**Review by Atmosfearful — CHANGES_REQUESTED** [review]
Oh wow @0xAeterno these showcase pages are great! Let's use these instead. Sorry I somehow glossed over that ticket you made!

I left a comment here with very slight tweak to your suggestions #630 . Feel free to add them to this PR @goatsOnDrugs

**Comment by goatsOnDrugs** [comment]
updated the urls and copy based on the comment in https://github.com/KlimaDAO/klimadao/issues/630

**Review by 0xAeterno — APPROVED** [review]
Awesome stuff, thanks @goatsOnDrugs :)
It's great to see these links updated 🔥

**Review by Atmosfearful — APPROVED** [review]

**Pull request merged**